core/src/main/java/tech/tablesaw/columns/Column.java

```diff
@@ -80,7 +81,9 @@ default int countUnique() {
   /**
    * Returns a string representation of the value at the given row.
    *
-   * @param row The index of the row.
+   * @param row
```

Comment:
can you revert the formatting changes in this file?

If you run `mvn compile` one time it will install a pre-commit hook so that when you run `git commit` it will automatically format the code to match the rest of the codebase

Reply:
Sure!
My bad, I forgot I got my IDE configured to format the code when I save.